src/main/java/org/generationcp/middleware/dao/ims/TransactionDAO.java

```diff
@@ -545,4 +557,217 @@ private void createTransactionRow(final List<TransactionReportRow> transactionRe
 		}
 
 	}
+
+	//New inventory functions, please locate them below this line to help cleaning in the near future.
+	private final String SEARCH_TRANSACTIONS_QUERY = "SELECT " //
+		+ "    act.trnid AS transactionId,"//
+		+ "    users.uname AS user,"//
+		+ "    (CASE"//
+		+ "        WHEN trnstat = 1 AND trnqty >= 0 THEN 'Deposit'"//
```

Comment:
@vmaletta we should use TransactionStatus and TransactionType enum in the new queries. Please fix SEARCH_TRANSACTIONS_QUERY and appends

Reply:
Ok